lib/matplotlib/tests/test_category.py

```diff
@@ -187,6 +187,65 @@ def test_plot_1d_missing(self):
         self.axis_test(ax.yaxis, self.dmticks, self.dmlabels, self.dmunit_data)
 
     @cleanup
+    @pytest.mark.usefixtures("data")
+    def test_plot_bytes(self):
+        counts = np.array([4, 6, 5, 1])
+        fig, ax = plt.subplots(ncols=3)
+
+        ax[0].bar(self.d, counts)
+
+        types = [v.encode('ascii') for v in self.d]
```

Comment:
kinda confused by this since self.d is ['a', 'c', 'c', 'd']...what's the point of the reencoding